build.gradle

```diff
@@ -22,89 +22,17 @@ repositories {
     jcenter()
 }
 
-apply plugin: 'gradle-flexversion'
-flexversion {
-    useTags = true
-}
-group = 'com.palantir'
-version = flexVersion()
-description = 'A Gradle plugin for applying Baseline-recommended build and IDE settings'
-
-dependencies {
-    compile gradleApi()
-    compile localGroovy()
-    compile 'com.google.guava:guava:18.0'
-
-    testCompile 'com.netflix.nebula:nebula-test:2.2.1'
-    testCompile 'junit:junit:4.11'
-    testCompile 'net.lingala.zip4j:zip4j:1.3.2'
-    testCompile 'org.hamcrest:hamcrest-all:1.3'
-    testCompile 'org.mockito:mockito-core:1.10.19'
-}
-
-task sourceJar(type: Jar) {
-    from sourceSets.main.allSource
-    classifier 'sources'
-}
-
-publishing {
-    publications {
-        PluginPublication(MavenPublication) {
-            from components.java
-
-            artifact(sourceJar) {
-                classifier 'sources'
-            }
-        }
-    }
-}
-
-bintray {
-    user = System.getenv('BINTRAY_USER')
-    key = System.getenv('BINTRAY_KEY')
-    pkg {
-        repo = 'releases'
-        name = 'gradle-baseline'
-        userOrg = 'palantir'
-        licenses = ['Apache-2.0']
-        publications = ['PluginPublication']
-    }
-}
-
-bintrayUpload.onlyIf {
-    project.version.isTag == true
-}
-
-bintrayUpload.dependsOn { generatePomFileForPluginPublicationPublication }
-bintrayUpload.dependsOn { sourceJar }
-bintrayUpload.dependsOn { build }
-
-pluginBundle {
-      website = 'https://github.com/palantir/gradle-baseline'
-      vcsUrl = 'https://github.com/palantir/gradle-baseline'
-      description = 'Baseline Java is a collection of Gradle plugins for configuring code quality tools in builds and generated Eclipse/IntelliJ projects.'
-      tags = ['java', 'checkstyle', 'code quality', 'findbugs', 'eclipse', 'idea']
+subprojects {
+    apply plugin: 'com.jfrog.bintray'
+    bintrayUpload.onlyIf {
+	    project.version.isTag == true
```

Comment:
apparently Eclipse decided to use tabs for some of the indents locally, where they displayed normally. Will fix, then merge.